tests/testthat/helper.R

```diff
@@ -0,0 +1,58 @@
+transition_example_a <- function() {
```

Comment:
these are potentially helpful examples of building simple example components of a `config`